### PR TITLE
add Fastly::Dictionary#item

### PR DESCRIPTION
**Pull request by lanej**
Returns a specific `Fastly::DictionaryItem`

### CODE REVIEW

File: lib/fastly/client.rb [hunk 1]
```diff
@@ -15,7 +18,7 @@ def initialize(opts)
       @customer = opts.fetch(:customer, nil)
       @oldpurge = opts.fetch(:use_old_purge_method, false)
 
-      base    = opts.fetch(:base_url, 'https://api.fastly.com')
+      base    = opts.fetch(:base_url, DEFAULT_URL)
```
Comment (gschorkopf) [inline lib/fastly/client.rb]: 💯

**Comment by blithe** [comment]
👍

File: test/fastly/dictionary_test.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+describe Fastly::Dictionary do
+
+  let(:client)     { Fastly.new(user: 'test@example.com', password: 'password') }
+  let(:service_id) { SecureRandom.hex(6) }
+  let(:version)    {  1 }
+  let(:dictionary) { Fastly::Dictionary.new({id: SecureRandom.hex(6), service_id: service_id, version: 1}, client) }
+
+  before {
+    stub_request(:post, "#{Fastly::Client::DEFAULT_URL}/login").to_return(body: '{}', status: 200, headers: { 'Set-Cookie' => 'tasty!' })
```
Comment (gschorkopf) [inline test/fastly/dictionary_test.rb]: Eyeroll.

File: test/fastly/dictionary_test.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+      assert_nil dictionary.item('key')
+    end
+
+    it 'returns the corresponding dictionary item when present' do
+      item_key   = 'key'
+      item_value = 'value'
+
+      response_body = JSON.dump(
+        "dictionary_id" => dictionary.id,
```
Comment (gschorkopf) [inline test/fastly/dictionary_test.rb]: Any reason why you're using double quotes here and single quotes elsewhere?

Comment (lanej) [inline test/fastly/dictionary_test.rb]: copy pasta from the api docs 🍝

**Pull request merged**